scripts/generateAssetData/generateRelatedAssetIndex/generateRelatedAssetIndex.ts

```diff
@@ -140,15 +196,20 @@ const processRelatedAssetIds = async (
   }
 
   const relatedAssetsResult = await getRelatedAssetIds(assetId, assetData)
+  const manualRelatedAssetsResult = getManualRelatedAssetIds(assetId)
 
   // ensure empty results get added so we can use this index to generate distinct asset list
-  const { relatedAssetIds, relatedAssetKey } = relatedAssetsResult ?? {
-    relatedAssetIds: [],
-    relatedAssetKey: assetId,
-  }
+  const { relatedAssetIds, relatedAssetKey } = manualRelatedAssetsResult ??
```

Comment:
not super familiar with the zerion related assets, but would we want to combine manual and related (unique) just in case?

Reply:
That was my thinking originally as well, but in the end landed on it being more sane to either go full Zerion when they have the data, or our own when we do have it